lib/supply/client.rb

```diff
@@ -292,10 +292,34 @@ def clear_screenshots(image_type: nil, language: nil)
       raise result.error_message if result.error?
     end
 
+    def upload_obb(obb_file_path)
+      result = api_client.execute(
```

Comment:
this is not the right call at all.

The call should be something like https://github.com/google/google-api-ruby-client/blob/master/generated/google/apis/androidpublisher_v2/service.rb#L966